scipioninstaller/installer.py

```diff
@@ -159,7 +162,8 @@ def getInstallationCmd(scipionHome, dev, args):
             cmd += cmdfy("python -m scipion installb xmippDev -j %d" % nProcess)
 
     else:
-        cmd = cmdfy("pip install scipion-app")
+        cmd = cmdfy("pip install scipion-pyworkflow")
+        cmd += cmdfy("pip install scipion-app")
```

Comment:
why now scipion-pyworkflow is installed and not before. Even, why scipion-em is not installed here. Maybe it's obvious, but it sounds the bell...

Reply:
There is a problem when packing scipion-em. It imports a few constants in the setup.py and triggers the import of pywokflow that is a requirement. We need to fix this, but meanwhile this will fix it.